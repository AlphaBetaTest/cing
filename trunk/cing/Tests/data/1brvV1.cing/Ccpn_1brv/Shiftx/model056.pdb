REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       4.943   0.360   3.637  1.00  0.00              
ATOM      2  CA  VAL A 171       4.491  -0.565   2.547  1.00  0.00              
ATOM      3  HA  VAL A 171       4.439   0.104   1.689  1.00  0.00              
ATOM      4  CB  VAL A 171       5.647  -1.593   2.278  1.00  0.00              
ATOM      5  HB  VAL A 171       6.550  -0.974   2.116  1.00  0.00              
ATOM      6  CG1 VAL A 171       5.913  -2.594   3.428  1.00  0.00              
ATOM      7 HG11 VAL A 171       6.108  -2.075   4.385  1.00  0.00              
ATOM      8 HG12 VAL A 171       5.051  -3.267   3.595  1.00  0.00              
ATOM      9 HG13 VAL A 171       6.795  -3.231   3.230  1.00  0.00              
ATOM     10  CG2 VAL A 171       5.539  -2.405   0.970  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.356  -1.750   0.102  1.00  0.00              
ATOM     12 HG22 VAL A 171       6.468  -2.964   0.749  1.00  0.00              
ATOM     13 HG23 VAL A 171       4.719  -3.147   0.993  1.00  0.00              
ATOM     14  C   VAL A 171       3.021  -1.161   2.522  1.00  0.00              
ATOM     15  O   VAL A 171       2.679  -1.736   1.481  1.00  0.00              
ATOM     16  N   PRO A 172       2.076  -1.033   3.500  1.00  0.00              
ATOM     17  CD  PRO A 172       2.364  -0.544   4.851  1.00  0.00              
ATOM     18  CA  PRO A 172       0.684  -1.520   3.346  1.00  0.00              
ATOM     19  HA  PRO A 172       0.633  -2.416   2.694  1.00  0.00              
ATOM     20  CB  PRO A 172       0.387  -1.973   4.788  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.694  -1.983   5.021  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.738  -3.015   4.925  1.00  0.00              
ATOM     23  CG  PRO A 172       1.191  -1.037   5.701  1.00  0.00              
ATOM     24  HG2 PRO A 172       0.576  -0.180   6.024  1.00  0.00              
ATOM     25  HG3 PRO A 172       1.530  -1.544   6.624  1.00  0.00              
ATOM     26  HD2 PRO A 172       2.435   0.563   4.822  1.00  0.00              
ATOM     27  HD3 PRO A 172       3.326  -0.950   5.211  1.00  0.00              
ATOM     28  C   PRO A 172      -0.266  -0.411   2.770  1.00  0.00              
ATOM     29  O   PRO A 172      -0.748   0.463   3.501  1.00  0.00              
ATOM     30  N   CYS A 173      -0.556  -0.461   1.456  1.00  0.00              
ATOM     31  H   CYS A 173      -0.003  -1.122   0.894  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.505   0.464   0.759  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.100   1.477   0.752  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.482   0.053  -0.719  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.965  -0.898  -0.876  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.502  -0.012  -1.072  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.526   1.121  -1.799  1.00  0.00              
ATOM     38  C   CYS A 173      -2.934   0.639   1.388  1.00  0.00              
ATOM     39  O   CYS A 173      -3.513   1.727   1.294  1.00  0.00              
ATOM     40  N   SER A 174      -3.463  -0.394   2.069  1.00  0.00              
ATOM     41  H   SER A 174      -2.820  -1.197   2.120  1.00  0.00              
ATOM     42  CA  SER A 174      -4.682  -0.270   2.931  1.00  0.00              
ATOM     43  HA  SER A 174      -5.517   0.017   2.298  1.00  0.00              
ATOM     44  CB  SER A 174      -4.998  -1.659   3.535  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.965  -1.619   4.076  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.147  -2.403   2.732  1.00  0.00              
ATOM     47  OG  SER A 174      -3.975  -2.116   4.425  1.00  0.00              
ATOM     48  HG  SER A 174      -3.933  -1.491   5.158  1.00  0.00              
ATOM     49  C   SER A 174      -4.673   0.833   4.043  1.00  0.00              
ATOM     50  O   SER A 174      -5.722   1.340   4.443  1.00  0.00              
ATOM     51  N   THR A 175      -3.479   1.181   4.524  1.00  0.00              
ATOM     52  H   THR A 175      -2.688   0.695   4.075  1.00  0.00              
ATOM     53  CA  THR A 175      -3.248   2.236   5.551  1.00  0.00              
ATOM     54  HA  THR A 175      -4.158   2.320   6.179  1.00  0.00              
ATOM     55  CB  THR A 175      -2.099   1.751   6.491  1.00  0.00              
ATOM     56  HB  THR A 175      -2.086   2.419   7.376  1.00  0.00              
ATOM     57  OG1 THR A 175      -0.827   1.847   5.854  1.00  0.00              
ATOM     58  HG1 THR A 175      -0.867   1.319   5.043  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.251   0.295   6.983  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.259   0.122   7.407  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.148  -0.421   6.142  1.00  0.00              
ATOM     62 HG23 THR A 175      -1.493   0.025   7.731  1.00  0.00              
ATOM     63  C   THR A 175      -3.013   3.674   4.961  1.00  0.00              
ATOM     64  O   THR A 175      -3.284   4.667   5.640  1.00  0.00              
ATOM     65  N   CYS A 176      -2.505   3.796   3.715  1.00  0.00              
ATOM     66  H   CYS A 176      -2.555   2.944   3.151  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.383   5.079   2.990  1.00  0.00              
ATOM     68  HA  CYS A 176      -1.927   5.840   3.654  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.394   4.734   1.863  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.748   3.993   1.150  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.217   5.587   1.192  1.00  0.00              
ATOM     72  SG  CYS A 176       0.176   4.211   2.584  1.00  0.00              
ATOM     73  C   CYS A 176      -3.761   5.653   2.520  1.00  0.00              
ATOM     74  O   CYS A 176      -3.992   6.858   2.621  1.00  0.00              
ATOM     75  N   GLU A 177      -4.654   4.785   2.000  1.00  0.00              
ATOM     76  H   GLU A 177      -4.239   3.892   1.723  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.083   5.080   1.701  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.362   4.277   1.001  1.00  0.00              
ATOM     79  CB  GLU A 177      -6.914   4.825   2.991  1.00  0.00              
ATOM     80  HB2 GLU A 177      -7.978   4.822   2.693  1.00  0.00              
ATOM     81  HB3 GLU A 177      -6.696   3.796   3.334  1.00  0.00              
ATOM     82  CG  GLU A 177      -6.747   5.798   4.191  1.00  0.00              
ATOM     83  HG2 GLU A 177      -5.691   5.852   4.511  1.00  0.00              
ATOM     84  HG3 GLU A 177      -7.014   6.828   3.891  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.579   5.421   5.411  1.00  0.00              
ATOM     86  OE1 GLU A 177      -6.977   4.468   6.172  1.00  0.00              
ATOM     87  OE2 GLU A 177      -8.667   5.916   5.683  1.00  0.00              
ATOM     88  C   GLU A 177      -6.462   6.370   0.919  1.00  0.00              
ATOM     89  O   GLU A 177      -7.407   7.097   1.234  1.00  0.00              
ATOM     90  N   GLY A 178      -5.728   6.576  -0.173  1.00  0.00              
ATOM     91  H   GLY A 178      -4.815   6.120  -0.089  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.865   7.788  -1.002  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.859   7.440  -2.045  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.842   8.258  -0.849  1.00  0.00              
ATOM     95  C   GLY A 178      -4.771   8.871  -0.865  1.00  0.00              
ATOM     96  O   GLY A 178      -4.808   9.852  -1.609  1.00  0.00              
ATOM     97  N   ASN A 179      -3.787   8.704   0.040  1.00  0.00              
ATOM     98  H   ASN A 179      -3.723   7.744   0.406  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.637   9.623   0.148  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.914  10.644  -0.183  1.00  0.00              
ATOM    101  CB  ASN A 179      -2.024   9.545   1.560  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.786   9.372   2.346  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.365   8.650   1.537  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.139  10.732   1.902  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.011  10.821   1.490  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.652  11.685   2.625  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.621  11.556   2.928  1.00  0.00              
ATOM    108 HD22 ASN A 179      -1.010  12.457   2.815  1.00  0.00              
ATOM    109  C   ASN A 179      -1.607   9.117  -0.846  1.00  0.00              
ATOM    110  O   ASN A 179      -0.943   8.095  -0.645  1.00  0.00              
ATOM    111  N   LEU A 180      -1.450   9.862  -1.914  1.00  0.00              
ATOM    112  H   LEU A 180      -2.060  10.679  -1.996  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.630   9.388  -3.021  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.048   8.379  -3.117  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.025  10.316  -4.175  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.181  11.350  -3.785  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.171  10.337  -4.817  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.206   9.892  -5.057  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.016   9.740  -4.346  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.596  11.013  -6.033  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.841  11.953  -5.505  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.785  11.242  -6.750  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.489  10.745  -6.628  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.958   8.593  -5.846  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.741   7.733  -5.186  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.838   8.306  -6.451  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.103   8.687  -6.543  1.00  0.00              
ATOM    128  C   LEU A 180       0.904   9.177  -2.803  1.00  0.00              
ATOM    129  O   LEU A 180       1.531   8.373  -3.490  1.00  0.00              
ATOM    130  N   ALA A 181       1.460   9.880  -1.822  1.00  0.00              
ATOM    131  H   ALA A 181       0.717  10.395  -1.349  1.00  0.00              
ATOM    132  CA  ALA A 181       2.824   9.639  -1.302  1.00  0.00              
ATOM    133  HA  ALA A 181       3.493   9.521  -2.166  1.00  0.00              
ATOM    134  CB  ALA A 181       3.257  10.878  -0.530  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.242  11.771  -1.176  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.600  11.046   0.344  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.287  10.736  -0.155  1.00  0.00              
ATOM    138  C   ALA A 181       2.935   8.356  -0.414  1.00  0.00              
ATOM    139  O   ALA A 181       3.834   7.548  -0.636  1.00  0.00              
ATOM    140  N   CYS A 182       2.026   8.158   0.567  1.00  0.00              
ATOM    141  H   CYS A 182       1.194   8.749   0.455  1.00  0.00              
ATOM    142  CA  CYS A 182       1.906   6.873   1.320  1.00  0.00              
ATOM    143  HA  CYS A 182       2.778   6.646   1.900  1.00  0.00              
ATOM    144  CB  CYS A 182       0.850   7.124   2.388  1.00  0.00              
ATOM    145  HB2 CYS A 182       0.981   8.100   2.863  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.138   7.163   1.934  1.00  0.00              
ATOM    147  SG  CYS A 182       0.826   5.794   3.597  1.00  0.00              
ATOM    148  C   CYS A 182       1.686   5.634   0.397  1.00  0.00              
ATOM    149  O   CYS A 182       2.346   4.599   0.507  1.00  0.00              
ATOM    150  N   LEU A 183       0.768   5.807  -0.547  1.00  0.00              
ATOM    151  H   LEU A 183       0.224   6.676  -0.461  1.00  0.00              
ATOM    152  CA  LEU A 183       0.517   4.886  -1.658  1.00  0.00              
ATOM    153  HA  LEU A 183       0.249   3.926  -1.237  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.712   5.495  -2.332  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.492   6.532  -2.607  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.828   5.028  -3.263  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.088   5.357  -1.679  1.00  0.00              
ATOM    158  HG  LEU A 183      -1.988   5.899  -0.721  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.117   6.029  -2.598  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.065   5.631  -3.631  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.147   5.863  -2.255  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.953   7.120  -2.667  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.503   3.882  -1.491  1.00  0.00              
ATOM    164 HD21 LEU A 183      -1.784   3.314  -0.878  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.481   3.779  -0.991  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.559   3.342  -2.456  1.00  0.00              
ATOM    167  C   LEU A 183       1.706   4.622  -2.639  1.00  0.00              
ATOM    168  O   LEU A 183       1.848   3.509  -3.140  1.00  0.00              
ATOM    169  N   SER A 184       2.546   5.636  -2.884  1.00  0.00              
ATOM    170  H   SER A 184       2.285   6.442  -2.298  1.00  0.00              
ATOM    171  CA  SER A 184       3.847   5.458  -3.611  1.00  0.00              
ATOM    172  HA  SER A 184       3.647   4.921  -4.560  1.00  0.00              
ATOM    173  CB  SER A 184       4.442   6.835  -3.987  1.00  0.00              
ATOM    174  HB2 SER A 184       4.647   7.439  -3.083  1.00  0.00              
ATOM    175  HB3 SER A 184       5.427   6.695  -4.475  1.00  0.00              
ATOM    176  OG  SER A 184       3.595   7.556  -4.883  1.00  0.00              
ATOM    177  HG  SER A 184       2.768   7.737  -4.406  1.00  0.00              
ATOM    178  C   SER A 184       4.904   4.577  -2.842  1.00  0.00              
ATOM    179  O   SER A 184       5.743   3.936  -3.479  1.00  0.00              
ATOM    180  N   LEU A 185       4.878   4.548  -1.492  1.00  0.00              
ATOM    181  H   LEU A 185       4.055   5.025  -1.085  1.00  0.00              
ATOM    182  CA  LEU A 185       5.705   3.594  -0.678  1.00  0.00              
ATOM    183  HA  LEU A 185       6.717   3.508  -1.094  1.00  0.00              
ATOM    184  CB  LEU A 185       5.780   4.058   0.810  1.00  0.00              
ATOM    185  HB2 LEU A 185       4.796   3.853   1.277  1.00  0.00              
ATOM    186  HB3 LEU A 185       6.486   3.400   1.353  1.00  0.00              
ATOM    187  CG  LEU A 185       6.134   5.533   1.104  1.00  0.00              
ATOM    188  HG  LEU A 185       5.385   6.141   0.557  1.00  0.00              
ATOM    189  CD1 LEU A 185       5.999   5.837   2.604  1.00  0.00              
ATOM    190 HD11 LEU A 185       4.984   5.607   2.978  1.00  0.00              
ATOM    191 HD12 LEU A 185       6.713   5.252   3.214  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.182   6.906   2.821  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.545   5.912   0.628  1.00  0.00              
ATOM    194 HD21 LEU A 185       7.657   5.768  -0.463  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.773   6.975   0.828  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.329   5.307   1.121  1.00  0.00              
ATOM    197  C   LEU A 185       5.215   2.118  -0.736  1.00  0.00              
ATOM    198  O   LEU A 185       6.017   1.181  -0.665  1.00  0.00              
ATOM    199  N   CYS A 186       3.894   1.931  -0.816  1.00  0.00              
ATOM    200  H   CYS A 186       3.375   2.783  -1.029  1.00  0.00              
ATOM    201  CA  CYS A 186       3.275   0.627  -0.949  1.00  0.00              
ATOM    202  HA  CYS A 186       3.968  -0.081  -0.476  1.00  0.00              
ATOM    203  CB  CYS A 186       1.991   0.734  -0.118  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.476  -0.212  -0.176  1.00  0.00              
ATOM    205  HB3 CYS A 186       2.064   0.934   0.945  1.00  0.00              
ATOM    206  SG  CYS A 186       0.889   1.950  -0.703  1.00  0.00              
ATOM    207  C   CYS A 186       2.958   0.153  -2.408  1.00  0.00              
ATOM    208  O   CYS A 186       3.187   0.819  -3.420  1.00  0.00              
ATOM    209  N   HIS A 187       2.391  -1.056  -2.454  1.00  0.00              
ATOM    210  H   HIS A 187       2.705  -1.578  -1.636  1.00  0.00              
ATOM    211  CA  HIS A 187       1.992  -1.736  -3.708  1.00  0.00              
ATOM    212  HA  HIS A 187       2.205  -1.103  -4.583  1.00  0.00              
ATOM    213  CB  HIS A 187       2.887  -3.000  -3.884  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.623  -3.726  -3.126  1.00  0.00              
ATOM    215  HB3 HIS A 187       2.623  -3.477  -4.829  1.00  0.00              
ATOM    216  CG  HIS A 187       4.404  -2.809  -3.818  1.00  0.00              
ATOM    217  ND1 HIS A 187       5.066  -2.860  -2.603  1.00  0.00              
ATOM    218  CD2 HIS A 187       5.061  -1.955  -4.702  1.00  0.00              
ATOM    219  CE1 HIS A 187       6.037  -1.929  -2.845  1.00  0.00              
ATOM    220  NE2 HIS A 187       6.161  -1.374  -4.091  1.00  0.00              
ATOM    221  HD2 HIS A 187       4.517  -1.552  -5.537  1.00  0.00              
ATOM    222  HE1 HIS A 187       6.443  -1.419  -1.983  1.00  0.00              
ATOM    223  C   HIS A 187       0.437  -1.935  -3.723  1.00  0.00              
ATOM    224  O   HIS A 187      -0.274  -1.051  -4.208  1.00  0.00              
ATOM    225  N   ILE A 188      -0.117  -3.054  -3.207  1.00  0.00              
ATOM    226  H   ILE A 188       0.554  -3.734  -2.837  1.00  0.00              
ATOM    227  CA  ILE A 188      -1.596  -3.313  -3.194  1.00  0.00              
ATOM    228  HA  ILE A 188      -2.140  -2.363  -3.318  1.00  0.00              
ATOM    229  CB  ILE A 188      -2.066  -4.200  -4.405  1.00  0.00              
ATOM    230  HB  ILE A 188      -1.471  -5.125  -4.343  1.00  0.00              
ATOM    231  CG2 ILE A 188      -3.572  -4.587  -4.326  1.00  0.00              
ATOM    232 HG21 ILE A 188      -3.818  -5.129  -3.394  1.00  0.00              
ATOM    233 HG22 ILE A 188      -4.233  -3.702  -4.375  1.00  0.00              
ATOM    234 HG23 ILE A 188      -3.874  -5.264  -5.145  1.00  0.00              
ATOM    235  CG1 ILE A 188      -1.796  -3.540  -5.793  1.00  0.00              
ATOM    236 HG12 ILE A 188      -2.418  -2.629  -5.901  1.00  0.00              
ATOM    237 HG13 ILE A 188      -0.760  -3.157  -5.820  1.00  0.00              
ATOM    238  CD1 ILE A 188      -1.952  -4.452  -7.025  1.00  0.00              
ATOM    239 HD11 ILE A 188      -1.332  -5.364  -6.942  1.00  0.00              
ATOM    240 HD12 ILE A 188      -2.998  -4.775  -7.181  1.00  0.00              
ATOM    241 HD13 ILE A 188      -1.641  -3.931  -7.950  1.00  0.00              
ATOM    242  C   ILE A 188      -1.983  -3.912  -1.796  1.00  0.00              
ATOM    243  O   ILE A 188      -1.546  -5.009  -1.439  1.00  0.00              
ATOM    244  N   GLU A 189      -2.829  -3.197  -1.028  1.00  0.00              
ATOM    245  H   GLU A 189      -3.134  -2.320  -1.461  1.00  0.00              
ATOM    246  CA  GLU A 189      -3.371  -3.643   0.295  1.00  0.00              
ATOM    247  HA  GLU A 189      -3.832  -2.743   0.738  1.00  0.00              
ATOM    248  CB  GLU A 189      -4.537  -4.663   0.105  1.00  0.00              
ATOM    249  HB2 GLU A 189      -4.154  -5.541  -0.453  1.00  0.00              
ATOM    250  HB3 GLU A 189      -4.825  -5.076   1.092  1.00  0.00              
ATOM    251  CG  GLU A 189      -5.825  -4.139  -0.591  1.00  0.00              
ATOM    252  HG2 GLU A 189      -5.587  -3.765  -1.604  1.00  0.00              
ATOM    253  HG3 GLU A 189      -6.520  -4.982  -0.760  1.00  0.00              
ATOM    254  CD  GLU A 189      -6.574  -3.035   0.157  1.00  0.00              
ATOM    255  OE1 GLU A 189      -6.285  -1.846   0.088  1.00  0.00              
ATOM    256  OE2 GLU A 189      -7.590  -3.520   0.919  1.00  0.00              
ATOM    257  C   GLU A 189      -2.320  -4.055   1.375  1.00  0.00              
TER     258      GLU A 189 
END   
