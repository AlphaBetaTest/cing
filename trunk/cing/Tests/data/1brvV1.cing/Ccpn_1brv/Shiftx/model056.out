# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      120.3111
  171   V  CA      60.4443
  171   V  CB      33.6146
  171   V  C      175.3073
  171   V  H        8.0393
  171   V  HA       3.9521
  171   V  HB       2.1181
  171   V  HG11     0.9506
  171   V  HG12     0.9750
  171   V  HG13     0.9232
  171   V  HG21     0.9829
  171   V  HG22     1.0050
  171   V  HG23     0.9908
  172   P  CA      61.1174
  172   P  CB      30.6770
  172   P  C      175.3442
  172   P  HA       4.4059
  172   P  HB2      2.3137
  172   P  HB3      2.2686
  172   P  HD2      3.7120
  172   P  HD3      3.7689
  172   P  HG2      2.0214
  172   P  HG3      2.0472
  173   C  N      123.8424
  173   C  CA      60.2796
  173   C  CB      40.7448
  173   C  C      177.1922
  173   C  H        8.6591
  173   C  HA       4.1649
  173   C  HB2      3.0192
  173   C  HB3      3.1881
  174   S  N      114.7662
  174   S  CA      61.4632
  174   S  CB      62.2988
  174   S  C      175.9618
  174   S  H        8.2084
  174   S  HA       3.9572
  174   S  HB2      4.0136
  174   S  HB3      4.1252
  175   T  N      111.9293
  175   T  CA      63.8148
  175   T  CB      69.4335
  175   T  C      176.1764
  175   T  H        7.7972
  175   T  HA       4.2248
  175   T  HB       4.2373
  175   T  HG21     1.2084
  175   T  HG22     1.2080
  175   T  HG23     1.2086
  176   C  N      119.5283
  176   C  CA      58.2047
  176   C  CB      40.7549
  176   C  C      173.8405
  176   C  H        8.1798
  176   C  HA       4.3633
  176   C  HB2      3.1163
  176   C  HB3      3.2114
  177   E  N      116.3755
  177   E  CA      57.4680
  177   E  CB      28.4123
  177   E  C      175.3979
  177   E  H        9.0870
  177   E  HA       3.8529
  177   E  HB2      2.1279
  177   E  HB3      2.0445
  177   E  HG2      2.2178
  177   E  HG3      2.2289
  178   G  N      104.5341
  178   G  CA      46.1997
  178   G  C      173.6866
  178   G  H        8.3787
  178   G  HA    -666.0000
  178   G  HA2      3.9567
  178   G  HA3      3.9560
  179   N  N      118.8364
  179   N  CA      52.5327
  179   N  CB      39.3906
  179   N  C      175.3127
  179   N  H        8.1065
  179   N  HA       4.5857
  179   N  HB2      2.8906
  179   N  HB3      2.7226
  179   N  HD21     7.1221
  179   N  HD22     6.0341
  180   L  N      129.0540
  180   L  CA      57.2787
  180   L  CB      41.6329
  180   L  C      179.4290
  180   L  H        8.4766
  180   L  HA       4.0555
  180   L  HB2      1.7224
  180   L  HB3      1.6993
  180   L  HD11     0.9175
  180   L  HD12     0.9175
  180   L  HD13     0.9175
  180   L  HD21     0.2378
  180   L  HD22     1.7630
  180   L  HD23     0.8659
  180   L  HG       1.7024
  181   A  N      120.7786
  181   A  CA      55.1787
  181   A  CB      18.3321
  181   A  C      179.7369
  181   A  H        7.8972
  181   A  HA       3.9235
  181   A  HB1      1.3859
  181   A  HB2      1.3496
  181   A  HB3      1.3295
  182   C  N      116.6796
  182   C  CA      60.0721
  182   C  CB      43.4595
  182   C  C      175.8607
  182   C  H        7.9982
  182   C  HA       4.1512
  182   C  HB2      2.9956
  182   C  HB3      3.2146
  183   L  N      119.4123
  183   L  CA      57.9660
  183   L  CB      41.2198
  183   L  C      179.4938
  183   L  H        8.0590
  183   L  HA       3.8376
  183   L  HB2      1.8479
  183   L  HB3      1.7041
  183   L  HD11     0.9117
  183   L  HD12     0.9865
  183   L  HD13     0.9128
  183   L  HD21     0.3943
  183   L  HD22     1.0611
  183   L  HD23     0.8564
  183   L  HG       1.6738
  184   S  N      114.3158
  184   S  CA      61.3379
  184   S  CB      62.7451
  184   S  C      176.3600
  184   S  H        7.8192
  184   S  HA       4.2524
  184   S  HB2      4.0163
  184   S  HB3      3.9202
  185   L  N      121.7388
  185   L  CA      56.7023
  185   L  CB      41.6188
  185   L  C      174.7946
  185   L  H        7.5922
  185   L  HA       4.1597
  185   L  HB2      1.6658
  185   L  HB3      1.7159
  185   L  HD11     0.9130
  185   L  HD12     0.9166
  185   L  HD13     0.9158
  185   L  HD21     0.2484
  185   L  HD22     1.4852
  185   L  HD23     0.8694
  185   L  HG       1.7974
  186   C  N      112.6346
  186   C  CA      54.2877
  186   C  CB      44.1086
  186   C  C      171.4400
  186   C  H        7.5032
  186   C  HA       4.6736
  186   C  HB2      3.1292
  186   C  HB3      3.0230
  187   H  N      115.5861
  187   H  CA      55.5834
  187   H  CB      30.3417
  187   H  C      174.9536
  187   H  H        8.3144
  187   H  HA       4.3858
  187   H  HB2      3.0672
  187   H  HB3      3.1288
  187   H  HD2      5.8381
  187   H  HE1      7.0784
  188   I  N      125.9671
  188   I  CA      60.6741
  188   I  CB      41.0104
  188   I  C      175.9686
  188   I  H        8.1683
  188   I  HA       4.3294
  188   I  HB       1.8850
  188   I  HD11     0.8739
  188   I  HD12     0.8680
  188   I  HD13     0.8716
  188   I  HG12     1.2799
  188   I  HG13     0.6133
  188   I  HG21     0.9156
  188   I  HG22     0.9138
  188   I  HG23     0.9163
 *189   E  N      120.5459
 *189   E  CA      56.1213
 *189   E  CB      30.2518
 *189   E  C      171.9686
 *189   E  H        8.8356
 *189   E  HA       4.2118
 *189   E  HB2      1.7353
 *189   E  HB3      1.7645
 *189   E  HG2      2.1289
 *189   E  HG3      2.3755
